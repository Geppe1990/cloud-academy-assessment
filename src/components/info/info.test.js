import React from "react";
import Info from "./info";

const character = {
	id: 13,
	name: "Alien Googah",
	status: "unknown",
	species: "Alien",
	type: "",
	gender: "unknown",
	origin: {
		name: "unknown",
		url: ""
	},
	location: {
		name: "Earth (Replacement Dimension)",
		url: "https://rickandmortyapi.com/api/location/20"
	},
	image: "https://rickandmortyapi.com/api/character/avatar/13.jpeg",
	episode: [
		"https://rickandmortyapi.com/api/episode/31"
	],
	url: "https://rickandmortyapi.com/api/character/13",
	created: "2017-11-04T20:33:30.779Z"
}

it("Renders info without crashing", () => {
	shallow(<Info character={character}/>);
});

it('match the snapshot', () => {
	const wrapper = renderer.create(<Info character={character} />).toJSON();
	expect(wrapper).toMatchSnapshot();
});