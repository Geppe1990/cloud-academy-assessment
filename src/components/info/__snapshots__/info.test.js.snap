// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match the snapshot 1`] = `
<div
  className="card h-100"
>
  <img
    alt="Card image cap"
    className="card-img-top"
    src="https://rickandmortyapi.com/api/character/avatar/13.jpeg"
  />
  <div
    className="card-body"
  >
    <h5
      className="card-title"
    >
      Alien Googah
    </h5>
    <p
      className="card-text"
    >
      <b
        className="card-title text-muted"
      >
        Status:
      </b>
       
      unknown
    </p>
    <p
      className="card-text"
    >
      <b
        className="card-title text-muted"
      >
        Species:
      </b>
       
      Alien
    </p>
    <p
      className="card-text"
    >
      <b
        className="card-title text-muted"
      >
        Gender:
      </b>
       
      unknown
    </p>
    <p
      className="card-text"
    >
      <b
        className="card-title text-muted"
      >
        Origin:
      </b>
       
      unknown
    </p>
  </div>
</div>
`;
